test/controllers/muncher_controller_test.rb

```diff
@@ -0,0 +1,54 @@
+require "test_helper"
+
+describe MuncherController do
+
+  describe 'search' do
+
+    it "should succeed" do
+      get root_path
+      must_respond_with :success
+    end
+  end
+
+  describe 'index' do
+
+    it "should succeed with some matches to the query" do
+      VCR.use_cassette("recipe") do
+        get muncher_index_path
+        recipes = EdamamApiWrapper.list_recipes("chicken")
+        must_respond_with :ok
+      end
+      must_respond_with :success
+    end
+
+    it "should redirect to root path with nil or empty string" do
+      VCR.use_cassette("recipe") do
+        get muncher_index_path
+        recipes = EdamamApiWrapper.list_recipes("")
```

Comment:
Again this test is operating on the wrapper rather than the controller.  

Your test should look like:
```ruby
    it "should redirect to root path with nil or empty string" do
      VCR.use_cassette("recipe") do
        get muncher_index_path, params: {search: ""}
        must_respond_with :redirect
        must_redirect_to root_path
      end
    end
```